### PR TITLE
Signup view

### PR DESCRIPTION
**Pull request by Angham116**
#12 prepared signup view.

### CODE REVIEW

File: src/views/signup.hbs [hunk 1]
```diff
@@ -1,4 +1,75 @@
...
+         </label>
+      </div>
+      <input type="submit" value="Next" id="next1">
+      <p> You already have an account ? <a href="/login">Sign In</a> </p>
+   </div>
+   <div class="section2">
+      <div class="category">
+         <label> Category : </label>
+         <select class="categoryFields">
+            <option>  </option>
```
Comment (Fatmasiam) [inline src/views/signup.hbs]: the category id also must be variables like spec_id

File: src/views/signup.hbs [hunk 2]
```diff
@@ -1,4 +1,75 @@
...
+            <option id="webDeveloper"> Web Developer </option>
+            <option id="uxUiDesigner"> UX/UI Designer </option>
+            <option id="translator"> Translator </option>
+            <option id="systemAnalyst"> System Analyst </option>
+         </select>
+      </div>
+      <div class="specialization">
+         <label> Specialization : </label>
+         <select class="specializationFields">
+            <option> </option>
```
Comment (Fatmasiam) [inline src/views/signup.hbs]:
```javascript
 {{#each specalization}}
            <option id="{{this.id}}"> {{this.name}}</option>
 {{/each}}
```

Comment (Angham116) [reply src/views/signup.hbs]: done.

**Review by ahmedalmi95 — APPROVED** [review]

File: src/views/signup.hbs [hunk 3]
```diff
@@ -1,4 +1,72 @@
-<div>
-    hello from sign up
+<div class="signUpContainer">
+   <h1>Sign Up</h1>
+   <form id="section1" class="section1">
```
Comment (kannanhassouna) [inline src/views/signup.hbs]: use descriptive names 
```suggestion
   <form id="section1" class="personal-info-section">
```

File: src/views/signup.hbs [hunk 4]
```diff
@@ -1,4 +1,72 @@
...
+      <div class="firstname">
+         <label for="firstname">First Name :
```
Comment (kannanhassouna) [inline src/views/signup.hbs:5]: ``` for ``` property look for ids and you don't have a input with ``` firstname ``` id

File: src/views/signup.hbs [hunk 5]
```diff
@@ -1,4 +1,72 @@
...
+         <input id="userfirstname" class="userfirstname" type="text" id="firstname" placeholder="Enter Your First Name" required>
```
Comment (kannanhassouna) [inline src/views/signup.hbs]:
```suggestion
         <input id="firstname" class="userfirstname" type="text"placeholder="Enter Your First Name" required>
```

File: src/views/signup.hbs [hunk 6]
```diff
@@ -1,4 +1,72 @@
...
+         </label>
+      </div>
+      <div class="lastname">
+         <label for="lastname">Last Name :
+         <input id="userlirstname" class="userlirstname" type="text" name='lastname' id="lastname" placeholder="Enter Your Last Name" required>
```
Comment (kannanhassouna) [inline src/views/signup.hbs]: same problem here

File: src/views/signup.hbs [hunk 7]
```diff
@@ -1,4 +1,72 @@
...
+         </label>
+      </div>
+      <div class="mobile">
```
Comment (kannanhassouna) [inline src/views/signup.hbs]: instead of repeating your  input each time you can use the power of partials

File: src/views/signup.hbs (hunk 5 above)
Comment (kannanhassouna) [reply src/views/signup.hbs]: sorry, I don't notice that but you have another problem, why you use id twice ?

File: src/views/signup.hbs [hunk 8]
```diff
@@ -1,4 +1,72 @@
...
+         <label for="mobile"> Mobile NO. : 
+         <input class="fields" type="text" name='mobile' id="mobile" placeholder="Enter Your Mobile Number" required>
+         </label>
+      </div>
+      <div class="email">
+         <label for="email">E-mail : 
+         <input class="userEmail" type="text" name='email' id="email" placeholder="example@hotmail.com" required>
```
Comment (kannanhassouna) [inline src/views/signup.hbs]: Indentation !!

File: src/views/signup.hbs [hunk 9]
```diff
@@ -1,4 +1,72 @@
...
+         <label> Category : </label>
+         <select id="categoryFields" class="categoryFields">
+            {{#each categoryId}}
+            <option class="spec" value="{{this.id}}"> {{this.name}}</option>
+            {{/each}} 
+         </select>
+      </div>
+      <div class="specialization">
+         <label> Specialization : </label>
+         <select id="specializationFields" class="specializationFields">
```
Comment (kannanhassouna) [inline src/views/signup.hbs]: be short and descriptive in naming 
```suggestion
         <select id="spec-fields" class="spec-fields">
```

File: src/views/signup.hbs [hunk 10]
```diff
@@ -1,4 +1,72 @@
...
+            {{#each specId}} 
+            <option class="spec" value="{{this.id}}"> {{this.name}}</option> 
+            {{/each}}
+         </select>
+      </div>
+      <div class="socialUrl">
+         <label for="userUrl">LinkedIn, GitHub, etc. profile URL : 
+         <input id="userUrl" class="userUrl" type="text" name='userUrl' id="userUrl" placeholder="Example: https://github.com/Someone" required>
```
Comment (kannanhassouna) [inline src/views/signup.hbs]: why you repeated ``` user ``` in each input ?
i meant in naming

**Comment by tomduggan85** [comment]
@Angham116 Looks good 👍

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

**Comment by tomduggan85** [comment]
@Angham116 I accidentally closed this PR by clicking the wrong button 🤦‍♂️Sorry about that!

**Comment by Angham116** [comment]
@knanahassouna1 done 🖒

**Pull request merged**